rules/yarn/index.bzl

```diff
@@ -2,7 +2,7 @@ DEFAULT_CMD_TPL = """
 # NOTE: BazelBinResolverPlugin in docusaurus.config.js depends on ROOTDIR being set
 # to the original execution working directory.
 export ROOTDIR=$$(pwd) &&
-export PACKAGEDIR=$$(dirname $(location {package})) &&
+export PACKAGEDIR=\\$$BUILD_WORKSPACE_DIRECTORY/$$(dirname $(location {package})) &&
```

Comment:
hm why is the `\\` needed? (here and below)

Reply:
without the back slashes, Bazel would fail with

```
/bin/bash: line 1: BUILD_WORKSPACE_DIRECTORY: unbound variable
```

when it tries to replace the variable with relevant Make variables.

Escaping excludes the variable from the substitution.

Reply:
any idea why `$$PATH` doesn't need the `\\` but `$$BUILD_WORKSPACE_DIRECTORY` does?

Reply:
```
> cat   bazel-bin/website/start.sh

export PATH=/private/var/tmp/_bazel_sluongng/06e573a93bc2d6a9cad4ad41f00b4310/execroot/buildbuddy/external/yarn/bin/yarnpkg/bin:/private/var/tmp/_bazel_sluongng/06e573a93bc2d6a9cad4ad41f00b4310/execroot/buildbuddy/external/nodejs_darwin_arm64/bin/nodejs/bin:/bin:/usr/bin:/usr/local/bin &&
cd $BUILD_WORKSPACE_DIRECTORY/website &&
yarn install &&
yarn start
```

This is the output in `bazel-bin/website/start.sh` on my laptop.
The `$PATH` env var was replaced by Bazel's Makevars while the `$BUILD_WORKSPACE_DIRECTORY` is kept as a bash variable.

So it worked because PATH is a legal makevars 😅